src/Libraries/OrchardCore.Commerce.AddressDataType/Regions.cs

```diff
@@ -15,9 +15,11 @@ public static class Regions
             .GetCultures(CultureTypes.SpecificCultures)
             .Select(culture =>
             {
-                // This sometimes throws "CultureNotFoundException: Culture is not supported." exception on Linux only.
-                try { return new RegionInfo(culture.LCID); } // #spell-check-ignore-line
-                catch (CultureNotFoundException) { return null; }
+                // #spell-check-disable
+                // This sometimes throws "CultureNotFoundException: Culture is not supported." exception on Linux, or
+                // "ArgumentException: Customized cultures cannot be passed by LCID, only by name." on Windows.
+                try { return new RegionInfo(culture.LCID); }
+                catch { return null; }
```

Comment:
Did you managed to repro the [mentioned error](https://github.com/OrchardCMS/OrchardCore.Commerce/issues/280)?

I want to test this somehow to see if it really fixes it.

Reply:
I couldn't repro it, but `ArgumentException` is `catch`able so this is guaranteed to absorb it by definition.